GOVERNANCE.md

```diff
@@ -0,0 +1,15 @@
+# `jest` Project Governance
+
+## Maintainers
+
+- [@SimenB](https://github.com/SimenB)
+
+Maintainers are responsible for issue/PR triage, feature additions, maintenance, bugfixes, security fixes, releases, promoting existing contributors to maintainers, managing repo and CI configuration, etc.
```

Comment:
How does the promoting of existing contributors work? Do they have to request being promoted? It is _ad hoc_ ? Etc.

Reply:
[`470616c` (#14948)](https://github.com/jestjs/jest/pull/14948/commits/470616ce916e5dce49d1ffa9bea4ab4c5e38f694)